src/dmd/backend/cod2.c

```diff
@@ -2011,8 +2011,7 @@ void cdcond(CodeBuilder& cdb,elem *e,regm_t *pretregs)
 
         opcode = 0x81;
         switch (sz2)
-        {   case 1:     opcode--;
```

Comment:
This has been this way for 30 years. I know of no bugs it has caused.

Reply:
See 18534 and the first commit.